xmake/modules/package/tools/cmake.lua

```diff
@@ -133,6 +133,10 @@ function _get_cflags(package, opt)
         table.join2(result, opt.cxflags)
     end
     table.join2(result, _get_cflags_from_packagedeps(package, opt))
+    local vs_runtime = package:config("vs_runtime")
```

Comment:
please add `if package:is_plat("windows")`

Reply:
Isn't vs_runtime forced to nil on other platforms than windows?

Reply:
Currently yes, but it is best to judge it here.